src/sandstorm/app-index/indexer.c++

```diff
@@ -389,7 +394,7 @@ void Indexer::updateIndexInternal(kj::StringPtr outputFilename, kj::StringPtr ou
     kj::FdOutputStream(file.getFd()).write(text.begin(), text.size());
     file.finalize(kj::str(outputDir, "/", appEntry.first, ".json"));
 
-    if (approvedApps) {
+    if (!experimental) {
```

Comment:
Not sure I understand this block, but not necessarily a showstopper if it actually works.